lib/howl/buffer.moon

```diff
@@ -201,10 +201,24 @@ class Buffer extends PropertyObject
       if @config.ensure_newline_at_eof and not @text\match "#{@eol}$"
         @append @eol
 
+      local backup
+      if config.backup_files and @file.exists
```

Comment:
The option is not defined as being global, but here we only check the top level config? Either it should be global or `@config` should be used. It would be great with a spec exercising this new behaviour. It should be possible for instance to use a file in a temp directory and removed the directory before saving, and then verify that the backup file is present.